File: java/bypstest-api/src/com/wilutions/byps/test/api/remote/RemoteEnums.java
package com.wilutions.byps.test.api.remote;

import com.wilutions.byps.BRemote;
import com.wilutions.byps.RemoteException;
import com.wilutions.byps.test.api.enu.EnumPlanets;

/**
 * 
 */
public interface RemoteEnums extends BRemote {

	EnumPlanets getPlanet() throws RemoteException;
	
	void setPlanet(EnumPlanets planet) throws RemoteException;
}

File: csharp/bypstest-ser/src-ser/com/wilutions/byps/test/api/remote/RemoteEnums.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using com.wilutions.byps;

namespace com.wilutions.byps.test.api.remote
{
	
	// checkpoint com.wilutions.byps.gen.cs.GenRemoteClass:93
	public interface RemoteEnums : BRemote {
	
		com.wilutions.byps.test.api.enu.EnumPlanets GetPlanet();
		void GetPlanet(BAsyncResult<com.wilutions.byps.test.api.enu.EnumPlanets> asyncResult) ;
		Task<com.wilutions.byps.test.api.enu.EnumPlanets> GetPlanetAsync();
		
		void SetPlanet(com.wilutions.byps.test.api.enu.EnumPlanets planet);
		void SetPlanet(com.wilutions.byps.test.api.enu.EnumPlanets planet, BAsyncResult<Object> asyncResult) ;
		Task SetPlanetAsync(com.wilutions.byps.test.api.enu.EnumPlanets planet);
		
		
	}
}  // end namespace
